packages/tools/client-debugger/client-debugger-view/src/Menu.tsx

```diff
@@ -0,0 +1,73 @@
+/*!
```

Comment:
Can we move this module into the `components` directory? Organization in this package has slipped a bit, but I would like to strictly have whatever we deem to be our "root" entry-point components live in the root of "src" if possible.

Reply:
No problem, moving it.